tmp/lanelet2_extension/include/lanelet2_extension/utility/query.hpp

```diff
@@ -175,6 +175,9 @@ lanelet::ConstLineStrings3d getLinkedParkingSpaces(
 lanelet::ConstLanelets getLinkedLanelets(
   const lanelet::ConstPolygon3d & parking_lot, const lanelet::ConstLanelets & all_road_lanelets);
 
+// query all no detection area in lanelet2 map
+lanelet::ConstPolygons3d getAllNoDetectionArea(const lanelet::LaneletMapConstPtr & lanelet_map_ptr);
+
```

Comment:
[minor comment]

Maybe this function is better placed at L117-L122

https://github.com/autowarefoundation/autoware_common/blob/4b42e2a3e5396779be10f4cbdee753f533133a31/tmp/lanelet2_extension/include/lanelet2_extension/utility/query.hpp#L117-L122

Reply:
Thank you for your comments!
I fixed the position of the function.
https://github.com/autowarefoundation/autoware_common/pull/92/commits/99feb65928eb3d9bda1629909d023e005b9620f7